src/game_battlealgorithm.cpp

```diff
@@ -114,6 +114,9 @@ Game_BattleAlgorithm::AlgorithmBase::AlgorithmBase(Type ty, Game_Battler* source
 	source_restriction(RPG::State::Restriction(source->GetSignificantRestriction()))
 {
 	Reset();
+	if (source) {
```

Comment:
> Turn off defend status in AlgorithmBase constructor 

The source check is unnecessary. source is never null (there is a source-> access a few lines above btw)

Reply:
Removed it